extensions/wikia/WikiaMobile/js/topbar.js

```diff
@@ -121,11 +122,11 @@ function (
 	//end search setup
 
 	//profile/login setup
-	if (wkPrfTgl) {
+	if (loginButton && !$loginButton.hasClass('new-login')) {
```

Comment:
I agree with Damian. Also, we could use className.indexOf instead of classList just to make life easier for all the old Androiders and iOS-ers in the world, as the old systems don't support DOMTokenLists :)